codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/QuicheQuicSslEngine.java

```diff
@@ -283,14 +289,32 @@ private boolean isEmpty(byte[] arr) {
         }
 
         void handshakeFinished(byte[] id, String cipher, String protocol, byte[] peerCertificate,
-                               byte[][] peerCertificateChain, long creationTime, long timeout) {
+                               byte[][] peerCertificateChain, long creationTime, long timeout, boolean singleUse) {
             synchronized (QuicheQuicSslEngine.this) {
                 initPeerCerts(peerCertificateChain, peerCertificate);
                 this.id = id;
                 this.cipher = cipher;
                 this.protocol = protocol;
                 this.creationTime = creationTime * 1000L;
                 this.timeout = timeout * 1000L;
+                this.invalid = singleUse;
```

Comment:
When I run 0RTT example I can't see 0RTT packet with wireshark, it seems cause by singleUse=true, and so we delete session in cache.

Reply:
doh! Fixed